src/style.css

```
@@ -1,3 +1,74 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --active-color: rgba(0, 172, 220, 1);
+  --main-color: rgba(0, 0, 0, 1);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.logo_link {
+  height: 40px;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  min-width: 1024px;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav__link {
+  font-size: 12px;
+  line-height: 14px;
+  padding: 20px 5px;
+  color: var(--main-color);
+  text-decoration: none;
+  cursor: pointer;
+  display: block;
+  text-transform: uppercase;
+
+}
+
+.nav__item:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  align-items: center;
+
+}
+
+.is-active::after {
+  content: "";
+  width: 100%;
+  border-bottom: 4px solid var(--active-color) ;
+  position: absolute !important;
```

no needs to add improtant
```suggestion
  position: absolute;
```